languagetool-core/src/main/java/org/languagetool/rules/RuleMatch.java

```diff
@@ -602,22 +599,21 @@ public int hashCode() {
   /**
    * The language that the text might be in if the error limit has been reached.
    *
-   * @since 5.3
+   * @since 6.4
    */
-  @Nullable
-  public String getErrorLimitLang() {
-    return errorLimitLang;
+  public Map<String, Float> getNewLanguageMatches() {
```

Comment:
Should we list this as a breaking change or even keep the old method for a while, implement it based on the map and deprecate it?

Reply:
I can't imagine it being used anywhere other than directly in our code.